#! /usr/bin/env python
PACKAGE = "pole_detect"
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("minhue1", int_t, 0, "Minnimum hue", 0, 0, 20)
gen.add("maxhue1", int_t, 0, "Maximum hue", 5, 5, 40)
gen.add("minval1", int_t, 0, "Minnimum value", 0, 0, 255)
gen.add("maxval1", int_t, 0, "Maximum value", 255, 0, 255)
gen.add("minsat1", int_t, 0, "Minnimum saturation", 20, 0, 255)
gen.add("maxsat1", int_t, 0, "Maximum saturation", 255, 0, 255)
gen.add("minhue2", int_t, 0, "Minnimum hue", 175, 140, 175)
gen.add("maxhue2", int_t, 0, "Maximum hue", 180, 160, 180)
gen.add("minval2", int_t, 0, "Minnimum value", 0, 0, 255)
gen.add("maxval2", int_t, 0, "Maximum value", 255, 0, 255)
gen.add("minsat2", int_t, 0, "Minnimum saturation", 20, 0, 255)
gen.add("maxsat2", int_t, 0, "Maximum saturation", 255, 0, 255)

exit(gen.generate(PACKAGE, "pole_detect", "ColorParams"))
